plaso/parsers/winreg_plugins/networks.py

```diff
@@ -25,11 +25,11 @@ class WindowsRegistryNetworkEventData(events.EventData):
     ssid (str): SSID of the connection.
   """
 
-  DATA_TYPE = 'windows:registry:network'
+  DATA_TYPE = 'windows:registry:networklist'
```

Comment:
Let's not change the data type until we have a consistent definition of what these needs to look like. This can break downstream analysis processes, and doesn't provide anything for Plaso.

Reply:
ack